crates/re_viewport/src/viewport.rs

```diff
@@ -832,9 +831,12 @@ impl<'a, 'b> egui_tiles::Behavior<SpaceViewId> for TabViewer<'a, 'b> {
 
                 self.edited = true;
             }
-            EditAction::TabSelected => {
+            EditAction::TabSelected | EditAction::TileResized => {
                 self.edited = true;
             }
+            EditAction::TileDragged => {
+                // no synchronisation needed
```

Comment:
```suggestion
                // No synchronization needed, because TileDragged happens when a drag starts, so no tiles are actually modified. When the drag completes, then we get `TileDropped` and run the synchronization.
```